File: SportskiCentarRKS_API/SportskiCentarRKS_API/ViewModels/GetTerminiResultVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace SportskiCentarRKS_API.ViewModels
{
    public class GetTerminiResultVM
    {

        public List<Row> rows;
        public  class Row
        {
            public int TerminID { get; set; }
            public string nazivProstorije { get; set; }
            public string Datum { get; set; }
            public string Pocetak { get; set; }
            public string Kraj { get; set; }
            public float Cijena { get; set; }
            public bool Slobodan { get; set; }
        }
    }
}

File: Sportski_centar_androidStudio/app/src/main/java/com/example/sports_centar/data/GetTerminiResultVM.java
package com.example.sports_centar.data;

import java.io.Serializable;
import java.util.List;

public class GetTerminiResultVM implements Serializable {
    public List<Row> rows;
    public class Row implements Serializable
    {
        public int TerminID;
        public String nazivProstorije;
        public String Datum;
        public String Pocetak;
        public String Kraj;
        public String Cijena;
        public boolean Slobodan;
    }
}
